backend/tests/integration/legacy/test_views.py

```diff
@@ -85,15 +85,40 @@ def test_redirects_query_from_legacy_url_with_empty_query(
 def test_redirects_query_from_legacy_url_not_in_labs(
     api_client, db, es_clear, create_record, base_app
 ):
-    with patch.dict(
-        base_app.config, {"LEGACY_BASE_URL": "https://legacy.inspirehep.net"}
-    ):
+    with patch.dict(base_app.config, {"LEGACY_BASE_URL": "https://old.inspirehep.net"}):
```

Comment:
not needed anymore as you changed the global config.

Reply:
You said it was better to do it like this to make the tests independent from the actual config, right?

Reply:
👍  we don't have to patch it